roles/pulp/tasks/install.yml

```diff
@@ -41,6 +41,12 @@
       check_mode: False
       register: result
 
+    - name: Make sure {{ pulp_group }} group exists
```

Comment:
I don't if Ansible properly interpolates here or if you need to quote. Still a bit of magic to me.